projectmanagement/src/main/java/life/qbic/projectmanagement/application/ProjectInformationService.java

```diff
@@ -74,20 +69,22 @@ public List<ProjectPreview> queryPreview(String filter, int offset, int limit,
     return new ArrayList<>(previewList);
   }
 
-  private Optional<Experiment> loadActiveExperimentForProject(Project project) {
-    return project.activeExperiment()
-        .map(experimentId ->
-            experimentRepository.find(experimentId)
-                .orElseThrow(
-                    () -> new ProjectManagementException(
-                        "The active experiment does not exist anymore.")
-                    // should never happen; indicates dirty removal of experiment from db
-                ));
+  private Project loadProject(ProjectId projectId) {
+    return projectRepository.find(projectId).orElseThrow(() -> new ProjectManagementException(
+            "Project with id" + projectId.toString() + "does not exit anymore")
+        // should never happen; indicates dirty removal of project from db
+    );
+  }
+
+  private Experiment loadActiveExperimentForProject(Project project) {
+    return experimentRepository.find(project.activeExperiment()).orElseThrow(
+        () -> new ProjectManagementException("The active experiment does not exist anymore.")
+        // should never happen; indicates dirty removal of experiment from db
+    );
```

Comment:
What is the status with the routing? Should this code be here in this service?
Where do we want to load experiments?